packages/front-end/components/Radix/Dropdown.tsx

```diff
@@ -0,0 +1,78 @@
+// radix dropdown wrapper
+import { DropdownMenu as RadixDropdownMenu } from "@radix-ui/themes";
+// import colors from radix-ui themes
+import type { MarginProps } from "@radix-ui/themes/dist/cjs/props/margin.props";
+type AllowedChildren = React.ReactNode;
+
+type Props = {
+  trigger: React.ReactNode;
+  disabled?: boolean;
+  errorLevel?: "error" | "warning";
+  description?: string;
+  children: AllowedChildren;
+} & MarginProps;
+export function Dropdown({ trigger, children, ...containerProps }: Props) {
+  return (
+    <RadixDropdownMenu.Root {...containerProps}>
+      <RadixDropdownMenu.Trigger>{trigger}</RadixDropdownMenu.Trigger>
+      <RadixDropdownMenu.Content>{children}</RadixDropdownMenu.Content>
```

Comment:
Looking at the design, I think we always want the `variant` prop for content to be `"soft"`